src/components/Counter.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from 'react';
+import moment from 'moment';
+import uuid from 'uuid/v1';
+
+import Input from './Input';
+import Button from './Button';
+import FoodsList from './FoodsList';
+
+
+class Counter extends Component {
+    constructor(props) {
+        super(props);
+
+        this.onFoodInputChange = this.onFoodInputChange.bind(this);
+        this.onCalInputChange = this.onCalInputChange.bind(this);
+        this.onAddFoodItem = this.onAddFoodItem.bind(this);
+        this.onRemoveFoodItem = this.onRemoveFoodItem.bind(this);
+
+        this.state = {
+            currentFoodInputValue: '',
+            currentCalInputValue: '',
```

Comment:
`Cal` is cryptic. It can be more explicit `currentCaloriesInputValue` to have better readability.